front/cypress/e2e/proposal_page_action.cy.ts

```diff
@@ -19,17 +19,13 @@ describe('Initiative show page actions', () => {
           cy.get('#e2e-initiative-show');
           cy.acceptCookies();
         });
-    });
-
-    afterEach(() => {
-      cy.apiRemoveInitiative(initiativeId);
```

Comment:
Maybe `afterEach` could still be helpful. If this spec fails, it will clean the DB to its initial state (maybe needed for other specs).